### ISSUE TITLE
Is alpha loss calculated for all the pixels or just for certain pixels that are unknown in trimap

### ISSUE THREAD

**Issue by kartikwar** [MarcoForte/FBA_Matting#41]
Hi in the repo https://github.com/huochaitiantang/pytorch-deep-image-matting/ 

the alpha prediction loss is calculated only for pixels that are unknown in trimap

"""
wi = torch.zeros(trimap.shape)
wi[trimap == 128] = 1.
t_wi = wi.cuda()
unknown_region_size = t_wi.sum()
"""

while training are you doing something similar or is it for all pixels?

Sorry couldn't find this in paper or code

**Comment by kartikwar** [comment]
Also do you take average of pixels for computing loss or is it just sum of differences of alpha for each pixel ?
